dev.rdm.RDM.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id": "dev.rdm.RDM",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "rdm",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=home",
```

Comment:
does it really need to access the whole home directory?

Reply:
RDM needs access to HOME dir to be able to import/export settings, ssh keys, etc.

Reply:
where are the settings?

also can't is use the ssh agent instead?

Reply:
Users may have older RDM versions installed (settings by default stored in $HOME/.rdm) and RDM needs access to it to provide a seamless migration to Flatpak variant of RDM.

Reply:
`--filesystem=~/.rdm`